src/ApplicationCore/Entities/FraudProtectionApiModels/SharedEntities.cs

```diff
@@ -0,0 +1,346 @@
+        /// 	Zip code provided with address.
+        /// </summary>
+        public String ZipCode { get; set; }
+
+        /// <summary>
+        /// 	ISO country code�provided with address.�2 alpha country code, e.g., 'US'
+        /// </summary>
+        public String Country { get; set; }
+    }
+
+    /// <summary>
+    /// 	Payment instrument associated this event
+    /// </summary>
+    public class PaymentInstrument
+    {
+        /// <summary>
+        /// 	Identifier for the PI in merchant system, mastered by Merchant.
+        /// </summary>
+        public String MerchantPaymentInstrumentId { get; set; }
+
+        /// <summary>
+        /// 	Type of payment. See PaymentInstrumentType enum.
+        /// </summary>
+        [Required]
+        public String Type { get; set; }
+
+        /// <summary>
+        /// 	Date PI was first entered in merchant system.
+        /// </summary>
+        public DateTimeOffset? CreationDate { get; set; }
+
+        /// <summary>
+        /// 	Date PI was last updated in merchant system.
+        /// </summary>
+        public DateTimeOffset? UpdateDate { get; set; }
+
+        /// <summary>
+        /// 	Defines the state of the PI. See PaymentInstrumentState enum.
+        /// </summary>
+        public String State { get; set; }
+
+        /// <summary>
+        /// 	The payment type: Examples values Visa, Mastercard, Amex, ACH, SEPA, UnionPay, Inicis, MobileBillingCarrier, Discover, AllPay, JCB, DiscoverDiners
+        /// </summary>
+        public String CardType { get; set; }
+
+        /// <summary>
+        /// 	Name of the user of the PI. For CREDITCARD/DEBITCARD only
+        /// </summary>
+        public String HolderName { get; set; }
+
+        /// <summary>
+        /// 	For CREDITCARD/DEBITCARD only
+        /// </summary>
+        public String BIN { get; set; }
+
+        /// <summary>
+        /// 	Date PI was Expired in merchant system. For CREDITCARD/DEBITCARD only
+        /// </summary>
+        public String ExpirationDate { get; set; }
+
+        /// <summary>
+        /// 	For CREDITCARD/DEBITCARD only
+        /// </summary>
+        public String LastFourDigits { get; set; }
+
+        /// <summary>
+        /// 	Email associated with the PI. For PAYPAL only
+        /// </summary>
+        public String Email { get; set; }
+
+        /// <summary>
+        /// 	For PAYPAL only
+        /// </summary>
+        public String BillingAgreementId { get; set; }
+
+        /// <summary>
+        /// 	For PAYPAL only
+        /// </summary>
+        public String PayerId { get; set; }
+
+        /// <summary>
+        /// 	For PayPal only. A value that indicates whether PayPal has verified the payer. Possible values Verified, Unverified
+        /// </summary>
+        public String PayerStatus { get; set; }
+
+        /// <summary>
+        /// 	For PayPal only. A value that indicates whether PayPal has verified the payer�s address. Possible values Confirmed, Unconfirmed
```

Comment:
Random unknown unicode character here in "payer's"

Reply:
Thanks.  VS doesn't show it that way, so I'm glad you noticed it.